.projenrc.js

```diff
@@ -20,15 +16,15 @@ const project = new AwsCdkConstructLibrary({
   defaultReleaseBranch: "main",
   releaseEveryCommit: false,
   cdkDependenciesAsDeps: false,
+
+  publishToPypi: {
+    distName: "datadog-cdk-constructs",
+    module: "datadog_cdk_constructs",
```

Comment:
Okay so after some research and testing, I was able to get the this functionality:
A user will `pip install datadog-cdk-constructs` and `from datadog_cdk_constructs import Datadog`, apparently python has an issue with modules having hyphens

Reply:
@nhinsch When you get the change may you give it another look before I merge?